### PR TITLE
Geneva Exporter - Set metric exporter temporality to delta

### PR DESCRIPTION
**Pull request by mic-max**
Ask for maintainers @open-telemetry/dotnet-maintainers 
- Release version 1.2.6 of the Geneva Exporter
- Unlist version 1.2.5 of the Geneva Exporter from NuGet

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,16 @@
 
 ## Unreleased
 
-## 1.2.5 [2022-Apr-20]
+## 1.2.6 [2022-Apr-21]
+
+* Set GenevaMetricExporter temporality preference back to Delta.
+[xxx](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/xxx)
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:8]: Update the PR#

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,16 @@
...
+
+## 1.2.5 [2022-Apr-20] Broken
+
+Note: This release was broken due to the GenevaMetricExporter
+using a TemporalityPreference of Cumulative instead of Delta, it has been
+unlisted from NuGet.
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:14]: Maybe add the details of the PR which had the incorrect changes?

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md (hunk 1 above)
Comment (mic-max) [reply src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:8]: haha, yes! this two step process is easy to forget :D

**Review by utpilla — APPROVED** [review]
Approved with suggestions.

**Comment by cijothomas** [comment]
> Ask for maintainers @open-telemetry/dotnet-maintainers
> 
> * Release version 1.2.6 of the Geneva Exporter
> * Unlist version 1.2.5 of the Geneva Exporter from NuGet

image
done.

**Pull request merged**